test/regress/undefined-shifts.txt

```diff
@@ -0,0 +1,20 @@
+;;; TOOL: run-gen-wasm-bad
```

Comment:
Looks great but can we contribute this test upstream?

Reply:
we're not sure. the fix/bug is only relevant to multi-memory (so it would go into the multi-memory proposal), but the test we used as a reference (test/binary/bad-alignment.txt) doesn't appear to be in upstream? (at least we couldn't find it in third_party/testsuite. not that we can even update that at this time...)

Reply:
(we don't actually know how to upstream this test. run-gen-wasm-bad / gen-wasm.py seems to be a custom tool, we guess we could run it manually to generate the required invalid binary wasm...?)

Reply:
That sounds right -- it'll have to be a `(module binary...)` in the script language.